main.go

```diff
@@ -58,6 +59,8 @@ const (
 	upperBoundRecommendation         = "upperBound"
 	uncappedTargetRecommendation     = "uncappedTarget"
 	vpaNamespace                     = "vpa"
+	gardener_vpa                     = "gardener_vpa"
+	gstatus                          = "vpa-recommender.gardener.cloud/status"
```

Comment:
```suggestion
	gardenerVPARecommendationAnnnotationKey                          = "vpa-recommender.gardener.cloud/status"
```

`gstatus` name doesn't convey the meaning for the constant clearly